### PR TITLE
Announcment for v0.14.7

### PR DESCRIPTION
**Pull request by alerque**
Before I post this on the blog and it goes out over RSS, @ctrlcctrlv or @khaledhosny is there anything you'd like me to adjust about mentioning the bounties involved here (and in the upcomming v0.14.8 release)?

### CODE REVIEW

File: content/blog/release-v0.14.7.md
Comment (khaledhosny) [inline content/blog/release-v0.14.7.md:28]: As long as it does not involve any glyph shapes variation, so it is only for limited use until the larger variable fonts change lands.

Comment (ctrlcctrlv) [reply content/blog/release-v0.14.7.md:28]: Yeah, if Khaled's okay with it I'd actually probably hold off on the mention until next release because it's a lot more impressive to say we did it all than "we did one small thing" "ok now we did the rest"

Comment (khaledhosny) [reply content/blog/release-v0.14.7.md:28]: Yes, I’d rather keep it for the next release announcement indeed.

Comment (alerque) [reply content/blog/release-v0.14.7.md:28]: Fair enough since you both feel that way. I'll adjust, thanks for the feedback.

**Comment by ctrlcctrlv** [comment]
New version looks good to me @alerque :-)

Thanks for the nice writeup, and I don't think it's such a big deal to delay the announcement one release as SILE regularly makes releases. It's not like some other projects I contribute to where missing the window means six months or a year of delay. And that's mostly down to you also—thank you!

**Review by ctrlcctrlv — APPROVED** [review]

**Pull request merged**
